template/src/components/app/router/ScrollToTop.tsx

```diff
@@ -0,0 +1,12 @@
+import { useEffect } from "react";
+import { useLocation } from "react-router-dom";
+
+export default function ScrollToTop() {
```

Comment:
Sieht aus als sollte diese Component eigentlich ein custom hook sein.
Außerdem arrow functions präferieren

Reply:
So empfohlen von React Router:
https://v5.reactrouter.com/web/guides/scroll-restoration/scroll-to-top